src/api/lib/tasks/git_cop.rake

```diff
@@ -0,0 +1,8 @@
+desc 'Run Git-cop locally'
+task :git_cop do
+  puts "\nCopying configuration file into ~/.config/git-cop/configuration.yml"
+  sh "mkdir -p ~/.config/git-cop"
+  sh "cp ../../dist/git-cop_configuration.yml ~/.config/git-cop/configuration.yml"
```

Comment:
What if I already have this file and why is this user specific? Isn't it rather project specific?

Reply:
> What if I already have this file?

This will overwrite it. It is the only way to ensure that you are using the last version of the configuration file.

> why is this user specific? Isn't it rather project specific?

This comes from the gem, I asked the same: https://github.com/bkuhlmann/git-cop/issues/3

Reply:
nitpick: I would prefer to use  [ruby](https://ruby-doc.org/stdlib-1.9.3/libdoc/fileutils/rdoc/FileUtils.html) here (in a rake task), instead of bash.

Regarding the overwrite, how about a confirmation dialogue? We could check if the files differ and if so ask for confirmation before overwriting the old conf

Reply:
@bgeuken 

> I would prefer to use ruby here (in a rake task), instead of bash.

And how do you want to do this?

> Regarding the overwrite, how about a confirmation dialogue? We could check if the files differ and if so ask for confirmation before overwriting the old conf

But I would say that we want that the run configuration is always the one we have inside our project, so I really think all this is not needed and would be annoying.

Reply:
Personally I dont see any problem with using bash to copy the files and such.

Reply:
Alright, I can live with it I guess...